cluster/swarm/cluster.go

```diff
@@ -241,6 +241,19 @@ func (c *Cluster) RemoveNetwork(network *cluster.Network) error {
 		for _, engine := range c.engines {
 			engine.DeleteNetwork(network)
 		}
+	} else if strings.Contains(err.Error(), "Is the docker daemon running") {
+		log.Debug("Some of the original engines unreachable - Rmoving global network from the current engines...")
+		for _, engine := range c.engines {
+			e1 := engine.RemoveNetwork(network)
+			if e1 == nil {
+				for _, engine := range c.engines {
+					engine.DeleteNetwork(network)
+				}
+				err = nil
+				break
+			}
```

Comment:
Reasonable. We should abandon other unrelated engine's error when removing network.
Still some advice:
1.Log message confuses me: the word `some` is confusing, can we change it into "The original engine ..."
2. If all engines are lost, can we log more detailed info about "all engines are down" or something like that?